src/java/com/zimbra/qa/selenium/projects/ajax/pages/calendar/DialogConfirmationDeclineAppointment.java

```diff
@@ -25,7 +25,7 @@
 	public static class Locators {
 		public static final String LocatorDivID = "CONFIRM_DELETE_APPT_DIALOG";
 		public static final String LocatorDivCSS = "css=div#CONFIRM_DELETE_APPT_DIALOG";
-		public static final String DontNotifyOrganizerRadioButton = "css=div[id='CONFIRM_DELETE_APPT_DIALOG_content'] td label:contains('Don�t notify organizer')";
+		public static final String DontNotifyOrganizerRadioButton = "css=div[id='CONFIRM_DELETE_APPT_DIALOG_content'] td label:contains('Don\u2019t notify organizer')";
```

Comment:
Can we put ascii value for smart quote in some string variable and use it here? It will be easier for someone else understand.